CSS/style.css

```diff
@@ -0,0 +1,206 @@
+#blue {
+    background-color: rgb(4, 4, 90);
+    height: 105px;
+    display: flex;
+}
+#gray {
+    font-family: Verdana, sans-serif;
+    text-align: center;
+    margin: auto;
+    width: 28%;
+    height: 46px;
+    background-color: white;
+    border: 1.5px solid red;
+}
+#bluebox {
+    margin: 0%;
+    padding: 0%;
+    font-size: 20px;
+}
+#graybox {
+    font-size: 10px;
+    font-weight: bold;
+}
+.container{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr
+}
+.Container_One{
```

Comment:
Follow the consistent naming conventions

Reply:
<img width="226" alt="Screenshot 2023-02-14 110123" src="https://user-images.githubusercontent.com/124386275/218648003-4ad4f911-b6be-4f6e-83f9-d7d39bdda7ce.png">